libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Testing/TestActions/AssertReply.cs

```diff
@@ -45,6 +45,10 @@ public AssertReply([CallerFilePath] string path = "", [CallerLineNumber] int lin
         [JsonProperty("exact")]
         public bool Exact { get; set; } = true;
 
+        /// <summary>
+        /// Validates the activity.
+        /// </summary>
+        /// <param name="activity">The activity to be verified. </param>
         public override void ValidateReply(Activity activity)
```

Comment:
Could use <inheritDoc/> for these override methods.

Reply:
@chrimc62 I'm not sure I understand the feedback. What do you mean?

Reply:
If you are doing an override, you should make the documentation be:
/// <inheritDoc/>
This will inherit the based doc definition which is usually appropriate.

Reply:
updated.